src/components/TagFilter.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react'
+import styled from '@emotion/styled'
+import { Tag } from '../mocks/TagMock'
+
+interface ButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElement> {
+  active: boolean
+}
+
+interface TagFilterProps {
+  tags: Tag[]
+  activeFilters: number[]
+  toggleFilter: (tagId: number) => void
```

Comment:
```suggestion
  onToggleFilter: (tagId: number) => void
```

반환 타입이 void 형식이고 컴포넌트를 소비하는 상위 컴포넌트에 filter toggle를 bottom-up으로 통지해주는 역할이라면 `on`을 prefix로 추가해서 callback props를 네이밍하는걸 권장드립니다.